tools/windows-node-installer/test/e2e/aws_test.go

```diff
@@ -164,16 +148,12 @@ func tearDownInstance(t *testing.T, Msg string, terr error) {
 
 	if createdInstanceID != "" {
 		err := awsProvider.TerminateInstance(createdInstanceID)
-		if err != nil {
-			t.Errorf("error terminating instance during teardown, %v", err)
-		}
+		require.NoError(t, err, "error terminating instance during teardown")
```

Comment:
Tear down needs to be best effort. So please update appropriately in all places?